src/net_processing.h

```diff
@@ -94,6 +94,9 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
      */
     void Misbehaving(const NodeId pnode, const int howmuch, const std::string& message);
 
+    /** Whether this node ignores txs received over p2p  */
```

Comment:
style-nit in 2ebb63696b0bac6dd627551d44aa935b769d8bde: double space

Reply:
Fixed